components/eamxx/docs/developer/source_tree.md

```diff
@@ -1,48 +1,48 @@
-+ `docs`: Documentation for the SCREAM project, including design documents,
-  instructions for building and testing SCREAM, and this document.
++ `docs`: Documentation for the EAMxx project, including design documents,
+  instructions for building and testing EAMxx, and this document.
 + `extern`: Source for certain lightweight third-party libraries, embedded
   directly into the repo (and not imported as submodules).
 + `scripts`: Miscellaneous scripts that implement workflows for running tests
   and analyzing performance.
-+ `src`: All C++ source code (and any bridges to Fortran) for SCREAM are stored
++ `src`: All C++ source code (and any bridges to Fortran) for EAMxx are stored
   here. We describe the contents of this directory in greater detail below.
-+ `tests`: Implements standalone, end-to-end tests for various SCREAM
++ `tests`: Implements standalone, end-to-end tests for various EAMxx
   components (RRTMG, HOMME, P3, SHOC, etc).
 
 In addition, you'll notice the following files in `components/eamxx`:
 
-+ `CMakeLists.txt`: The CMake file that defines SCREAM's configuration/build
++ `CMakeLists.txt`: The CMake file that defines EAMxx's configuration/build
   system.
 + `CTestConfig.cmake`: This CTest file contains parameters that determine how
   our test results are reported to the [E3SM CDash Site](http://my.cdash.org/submit.php?project=E3SM).
-+ `README.md`: SCREAM's top-level README file, which describes the project and
++ `README.md`: EAMxx's top-level README file, which describes the project and
   its purpose.
 
 ## The `src` Directory
 
-Herein lіes the source code for SCREAM. Broadly, here's where things are:
+Herein lіes the source code for EAMxx. Broadly, here's where things are:
 
 + `control`: Contains the atmosphere driver and basic tests for it.
-+ `dynamics`: Here's where HOMME lives within SCREAM, along with code for
-  interfacing with it using SCREAM's data structures.
-+ `interface`: Glue code for embedding SCREAM within E3SM as an atmosphere
++ `dynamics`: Here's where HOMME lives within EAMxx, along with code for
+  interfacing with it using EAMxx's data structures.
++ `interface`: Glue code for embedding EAMxx within E3SM as an atmosphere
   component.
 + `physics`: Source code for physics-related atmospheric processes, including
-  + `p3`: The C++/Kokkos implementation of P3 microphysics within SCREAM.
-  + `shoc`: The C++/Kokkos implementation of SHOC macrophysics within SCREAM.
-  + `rrtmgp`: A stub for the radiation processes as represented in SCREAM.
+  + `p3`: The C++/Kokkos implementation of P3 microphysics within EAMxx.
+  + `shoc`: The C++/Kokkos implementation of SHOC macrophysics within EAMxx.
+  + `rrtmgp`: A stub for the radiation processes as represented in EAMxx.
   + `common`: Utilities and data structures common to these processes.
-+ `share`: Utilities used by various components within SCREAM. A lot of things
++ `share`: Utilities used by various components within EAMxx. A lot of things
```

Comment:
We should remove the line `A lot of things here will likely end up in ekat`.

Reply:
And we could probably add "Notably, the io subfolder contains the interfaces to the scorpio library"